### ISSUE TITLE
Connection field "port" data type check fails since incorrect argument "float64" is used

### ISSUE THREAD

**Issue by brain-buster** [airflow-helm/charts#136]
**What is the bug?**

I pass the connection port value using --set flag (for example 5432 for "postgres" type connection). Bash convert it to "int64", but in secret-job-scripts.yaml -> create_connections.py there is the following:

    {{- if .port }}
        {{- if not (typeIs "float64" .port) }}
        {{ required "each `port` in `airflow.connections` must be int-type!" nil }}
        {{- end }}
        port={{ .port }},

Of course it fails. I tested chart with '(typeIs "int64" .port)' instead of '(typeIs "float64" .port)' and it works.

**What are your Helm values?**

Command to pass connection in values.yaml (used 'template' instead 'install' or 'upgrade' to debug):

helm template airflow-stable/airflow --debug \
  --set airflow.connections[0].id="postgres_test_db" \
  --set airflow.connections[0].type="postgres" \
  --set airflow.connections[0].port=5432 \
  --set airflow.connections[0].host="localhost" \
  --set airflow.connections[0].login="user" \
  --set airflow.connections[0].password="password" \
  --set airflow.connections[0].schema="test_db"

**What is your Kubernetes Version?**:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-11T13:09:17Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

**What is your Helm version?**:

```console
$ helm version
version.BuildInfo{Version:"v3.3.3-rancher3", GitCommit:"657df59bbba1d9e175cf5080d4885bd57d037906", GitTreeState:"clean", GoVersion:"go1.13.15"}
```

**Comment by thesuperzapper** [comment]
@brain-buster can you confirm if this still happens when you pass a "custom-values.yaml" to helm install/upgrade?

Also, do you want to raise a PR to fix this?

**Comment by brain-buster** [comment]
No, if a port value is defined in "custom-values.yaml" then data type check will be successed, but it is not flexible to hardcoding connection params in "custom-values.yaml" - I am using GitLab variables to pass connection params.

**Comment by brain-buster** [comment]
@thesuperzapper I created PR as you asked me.
Please note that I extended field "port" data type check (now allowed: "int64" and "float64") since there are 2 cases:
  1. It we set port using "custom-values.yaml" -> it will be "float64"
  2. If we set port using --set flag -> if will be "int64"

**Comment by thesuperzapper** [comment]
@brain-buster this has been fixed in `8.0.7`

**Comment by brain-buster** [comment]
@thesuperzapper thank you!
